server/server.go

```diff
@@ -130,7 +130,7 @@ type Server struct {
 	inShutdownMode bool
 }
 
-// ConnectionCount gets current connection count.
+// ConnectionCount gets current connection capacity.
```

Comment:
According to the source code, it returns current connection count instead of capacity?